packages/frontend/src/components/topicsMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import { useHistory } from 'react-router-dom'
+
+const TopicsMenu = () => {
+    const history = useHistory()
+
+    const goToPhil = () => {
+        history.push('/philosophy', { isConfirmed: true })
+    }
+    const goToTech = () => {
+        history.push('/technology', { isConfirmed: true })
+    }
+    const goToPol = () => {}
+    const goToMus = () => {}
+    const goToBiz = () => {}
+    const goToLit = () => {}
+    const goToFit = () => {}
+    const goToPho = () => {}
+    const goToNews = () => {}
+
+    return (
+        <div className="topics-menu">
+            <div className="topic-buttons">
+                <div onClick={goToPhil}>Philosophy</div>
+                <div onClick={goToTech}>Technology</div>
+                <div>Politics</div>
+                <div>Music</div>
+                <div>Business</div>
+                <div>Literature</div>
+                <div>Fitness</div>
+                <div>Photography</div>
+                <div>News</div>
```

Comment:
can this be iterative?

Reply:
Good idea. Here's what I think so far:

```ts
const topics = [
  {
    name: 'Philosophy',
    onClick: goToPhil,
  },
  {
    name: 'Technology',
    onClick: goToTech,
  },
  {
    name: 'Politics',
  },
  {
    name: 'Music',
  },
  {
    name: 'Business',
  },
  {
    name: 'Literature',
  },
  {
    name: 'Fitness',
  },
  {
    name: 'Photography',
  },
  {
    name: 'News',
  },
];

const NavLinks = () => {
  return (
    <>
      {topics.map(topic => (
        <div key={topic.name} onClick={topic.onClick}>
          {topic.name}
        </div>
      ))}
    </>
  );
};
```

Would like to find a way to incorporate having a `Topics` enum and condense the `onClick` logic though.